src/Microsoft.AspNet.Server.Kestrel/Http/MessageBody.cs

```diff
@@ -3,29 +3,41 @@
 
 using System;
 using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
 using Microsoft.Framework.Primitives;
 
 namespace Microsoft.AspNet.Server.Kestrel.Http
 {
-    public abstract class MessageBody : MessageBodyExchanger
+    public abstract class MessageBody
```

Comment:
Can do, but nested classes are allowed, and they do come after all other MessageBody members